package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "apollo-trainning",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon  --exec babel-node  ./src"
+  },
+  "author": "pankaj sharma",
+  "license": "ISC",
+  "dependencies": {
+    "@babel/cli": "^7.5.5",
+    "@babel/core": "^7.5.5",
+    "@babel/node": "^7.5.5",
+    "apollo-server-express": "^2.8.1",
+    "dotenv": "^8.1.0",
+    "express": "^4.17.1",
+    "graphql": "^14.4.2",
+    "nodemon": "^1.19.1"
```

Comment:
please install nodemon and babel in devDependencies

Reply:
Done .I have set nodemon and babel in devDependencies